CHANGELOG

```diff
@@ -0,0 +1,17 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Upcoming]
+
+## [0.16.1] - 2019-06-24
```

Comment:
<!--
  1 failure:  The latest date i...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: CHANGELOG;
  Line: 9;
-->
- :no_entry_sign: The latest date in the CHANGELOG is not today!